test/suppose-process.test.js

```diff
@@ -143,7 +118,7 @@ describe('process', function()
     })
   })
 
-  it('should call a method as response', function(done)
+  it_posixOnly('should call a method as response', function(done)
```

Comment:
`ls` is not available on Windows. The test is for POSIX only. We can write one specifically for Windows.